tests/integration/externalproperties/pom.xml

```diff
@@ -42,6 +42,10 @@
             <artifactId>jersey-test-framework-provider-jetty</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
+            <artifactId>jaxb-impl</artifactId>
```

Comment:
I think you need this dependency only for test, right?. Then it should contain the test scope.

Reply:
Actually I am wrong. This is already in a test folder, so ignore this comment.